news/_posts/translation/2020-07-10-do-you-speak-prestashop-june-2020.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: post
+title:  "Do you speak PrestaShop? – June 2020 edition"
+subtitle: "Where you'll learn that Esperanto is on the rise"
+date:  2020-07-10 14:30:11
+authors: [ JuneByun ]
+icon: icon-world
+image: /assets/images/2019/01/Build-Crowdin-banner.jpg
+tags: [translation, i18n, l10n]
+---
+
+Contributing to PrestaShop is not only about the code, it's also about taking part in the [PrestaShop translation project](https://crowdin.com/project/prestashop-official)! This report will tell you how the translations of the software evolved in May.
+
+![Crowdin Monthly banner](/assets/images/2019/01/Build-Crowdin-banner.jpg)
+
+## Project news
+
+
+### First news, Esperanto is on the rise :fire:
+
+I am thrilled to let you know that Esperanto has been on the rise since the past month. From the beginning of May to the end of June, the Esperanto translation project went from 25% to 34% words translated, that is to say a 9% evolution. I wish to take this opportunity to thank mirmal and Milan Kolka (milankolka) for your work! :heart_eyes: 
```

Comment:
```suggestion
I am thrilled to let you know that Esperanto has been on the rise since the last past months. From the beginning of May to the end of June, the Esperanto translation project went from 25% to 34% words translated, that is to say a 9% evolution. I wish to take this opportunity to thank [mirmal](https://crowdin.com/profile/mirmal) and [Milan Kolka](https://crowdin.com/profile/milankolka) for your work! :heart_eyes: 
```

Reply:
I think you got confused with the numbers of June. On Sandrine, from May to June, it is more like it went from 9% to 34% which represents a 25% evolution!

<img width="758" alt="Capture d’écran 2020-07-08 à 09 33 54" src="https://user-images.githubusercontent.com/32565890/86890926-521bc800-c0fe-11ea-81af-5d59d8beac8b.png">

Reply:
Oops, indeed! Thank you for catching this mistake :p

Reply:
No prob. ;-)